1-js/01-getting-started/1-intro/article.md

```diff
@@ -27,15 +27,15 @@ Brauzerin daxilində "JavaScript virtual maşını" adlı bir mühərrik mövcud
 Müxtəlif mühərriklərin müxtəlif kodadları var. Misalçün:
 
 - [V8](<https://en.wikipedia.org/wiki/V8_(JavaScript_engine)>) -- Chrome və Operada.
-- [SpiderMonkey](https://en.wikipedia.org/wiki/SpiderMonkey) -- in Firefoxda.
+- [SpiderMonkey](https://en.wikipedia.org/wiki/SpiderMonkey) -- Firefoxda.
 - [Chakra](<https://en.wikipedia.org/wiki/Chakra_(JScript_engine)>) -- Microsoft Edgedə
 - [JavaScriptCore](https://en.wikipedia.org/wiki/WebKit#JavaScriptCore) -- Safaridə (WebKit) və s.
 
-Yuxarıdakı terminləri yadda saxlamağınız tövsiyyə olunur çünki, bu cür terminlər developer məqalələrində çox istifadə olunur. Nümunə üçün onu deyə bilərik ki, kimsə sizə "filan xüsusiyyət V8-də dəstəklənir" deyəndə, o dəqiqə biləcəksiniz ki, Chrome və Opera (bir də gördünüz MS Edge) bu xüsusiyyəti dəstəkləyir.
+Yuxarıdakı terminləri yadda saxlamağınız tövsiyyə olunur, çünki bu cür terminlər developer məqalələrində çox istifadə olunur. Nümunə üçün onu deyə bilərik ki, kimsə sizə "filan xüsusiyyət V8-də dəstəklənir" deyəndə, o dəqiqə biləcəksiniz ki, Chrome və Opera (bir də gördüyünüz MS Edge) bu xüsusiyyəti dəstəkləyir.
```

Comment:
çünki bu cür terminlər "proqramçılıq" məqalələrində çox istifadə olunur. - I think it is better to translate developer as we try to do whole translation.
deyəndə( there is no comma here) o dəqiqə biləcəksiniz ki, Chrome və Opera